### PR TITLE
add app-template

### PR DESCRIPTION
**Pull request by alexgbor**

### CODE REVIEW

File: staff/alexgbor/css-layout-simple-adaptive/index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" media="screen and (max-width: 768px)" href="styles/main-mobile.css">
```
Comment (MuyZan) [inline staff/alexgbor/css-layout-simple-adaptive/index.html:8]: TIP: do the css calling inside a main css with @import

Comment (alexgbor) [reply staff/alexgbor/css-layout-simple-adaptive/index.html:8]: Thanks!

Comment (mikelpmc) [reply staff/alexgbor/css-layout-simple-adaptive/index.html]: 👍

File: staff/alexgbor/count-words/scripts/countWords.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+            if (str[i]===" ") {
+                counter+=1;
+            }
+        }
+    return counter;
+    }
+    else {
+        throw Error("You must input a string.");
+    }
+}
```
Comment (mikelpmc) [inline staff/alexgbor/count-words/scripts/countWords.js]: Tip: To count words yo could split the string into an array to have each word in an array position and then get the length of that array.

[split](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split)

Comment (alexgbor) [reply staff/alexgbor/count-words/scripts/countWords.js]: Correct, that's shorter and more elegant. Just tested it in the console and works flawlessly, give me a couple of minutes to re-push it. Thanks!

Comment (alexgbor) [reply staff/alexgbor/count-words/scripts/countWords.js]: Done. 👍

**Review by mikelpmc — APPROVED** [review]
👍

**Review by agandia9 — COMMENTED** [review]
ok!

**Pull request merged**